app/components/primer/beta/button.rb

```diff
@@ -68,7 +68,10 @@ class Button < Primer::Component
       renders_one :trailing_visual, types: {
         icon: Primer::Beta::Octicon,
         label: Primer::Beta::Label,
-        counter: Primer::Beta::Counter
+        counter: lambda { |**system_arguments|
```

Comment:
Pls lmk if there is a better way to know that the `trailing_visual` is a counter, without setting this boolean.